editor/src/document/document_file.rs

```diff
@@ -82,7 +82,6 @@ pub struct DocumentMessageHandler {
 	#[serde(with = "vectorize_layer_metadata")]
 	pub layer_metadata: HashMap<Vec<LayerId>, LayerMetadata>,
 	layer_range_selection_reference: Vec<LayerId>,
-	#[serde(skip)]
```

Comment:
Was this intentional?

Reply:
Yes, this allows the document position to be restored from saved files as well as autosave